Dockerfile

```
@@ -1,31 +1,33 @@
-# Start by building the sta-rs library in a builder container.
-FROM rust:1.60 as rust-builder
+# Start by building the nitriding proxy daemon.
+FROM golang:1.20 as go-builder
```

This fetches the Docker image from Amazon's container registry instead of Docker's, which prevents the rate limits that we keep running into.
```suggestion
FROM public.ecr.aws/docker/library/golang:1.20 as go-builder
```